security/keycloak-oidc-client-reactive-extended/src/test/resources/logout.properties

```diff
@@ -22,4 +22,4 @@ quarkus.oidc.token-cache.max-size=1
 
 # PKCE
 quarkus.oidc.authentication.pkce-required=true
-quarkus.oidc.authentication.pkce-secret=eUk1p7UB3nFiXZGUXi0uph1Y9p34YhBU
\ No newline at end of file
+quarkus.oidc.authentication.stateSecret=eUk1p7UB3nFiXZGUXi0uph1Y9p34YhBU
```

Comment:
this is wrong, it is `state-secret` https://quarkus.io/guides/all-config#quarkus-oidc_quarkus-oidc-authentication-state-secret

Reply:
Thanks I was looking at Quarkus code and use the name from it not actual property.

> seems like we are not testing that property judging by CI results?

Looking at the docs and it seems that property is not needed to verify the pkce (https://github.com/quarkusio/quarkus/blob/main/extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java#L968).